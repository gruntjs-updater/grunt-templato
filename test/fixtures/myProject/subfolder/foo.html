<html>
<head>
    <title><%= fooTitle %></title>
</head>
<body>
    <ul>
        <li><%= list.one %></li>
        <li><%= list.two %></li>
    </ul>
</body>
</html>