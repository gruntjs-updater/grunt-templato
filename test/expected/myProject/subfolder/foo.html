<html>
<head>
    <title>My title</title>
</head>
<body>
    <ul>
        <li>1</li>
        <li>secondOne</li>
    </ul>
</body>
</html>